HHS/Head-Start-TTADP
# [TTAHUB-2118] Ensure FEI root cause is disabled if on on approved report with response

## Description of change

If we have an FEI goal on an approved report with at least one root cause set we should disable the field.

## How to test

- Review the code.
- Edit a FEI goal on a approved report with one root cause, FEI root cause should be disabled.
- Edit a FEI goal on a approved report without a root cause, FEI root cause should be enabled. 
- Edit a FEI goal on unapproved report with one root cause, FEI root cause should be enabled.  
- Edit a FEI goal on unapproved report with no root causes set, FEI root cause should be enabled.  

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2118


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
